src/components/Dashboard/EventCard.vue

```diff
@@ -204,7 +229,7 @@ function handleJoin({ call = false } = {}) {
 <style scoped lang="scss">
 .event-card {
 	position: relative;
-	height: 225px;
+	height: 250px;
```

Comment:
```suggestion
	height: 250px;
	background-color: var(--color-main-background);
```
Or it inherits transparent color =P
<img width="217" alt="image" src="https://github.com/user-attachments/assets/d925c253-e434-43a1-82fb-76ca5c4a336e" />